pytest_django/plugin.py

```diff
@@ -231,7 +238,7 @@ def pytest_load_initial_conftests(early_config, parser, args):
 
         # Forcefully load django settings, throws ImportError or
         # ImproperlyConfigured if settings cannot be loaded.
-        from django.conf import settings
+        from django.conf import settings  # noqa
```

Comment:
Hmm, should we use `django_settings` for the fixture then instead?  Makes it more explicit anyway.
The old name would still be available, but deprecated.